modules/node/src/admin/admin.service.ts

```diff
@@ -52,16 +54,25 @@ export class AdminService {
   /**  Get Channels by xpub */
 
   async getChannelStates(userPublicIdentifier: any): Promise<CFCoreRecord[]> {
-    // get channel by xpub
+    // get channel by xpubgetChannelStateByMultiSig
     return await this.channelService.getChannelState(userPublicIdentifier.id);
   }
 
+  async getChannelStateByMultiSig(multisigAddress: string): Promise<CFCoreRecord[]> {
+    // get channel by xpub
+    return await this.channelService.getChannelStateByMultiSig(multisigAddress);
+  }
+
```

Comment:
Tracing through the code path, this should return a `StateChannelJSON` object (once typings on the `getChannelStateByMultisig` fn in `channelService` has better typings)